shell/assets/translations/en-us.yaml

```diff
@@ -237,6 +237,25 @@ about:
     title: Image Lists
   downloadCLI:
     title: CLI Downloads
+  diagnostic:
+    title: Diagnostic
+    checkboxTooltip: Get the response times when fetching data from the server for the top 10 highest count resources when downloading the diagnostic data. It may take a while
```

Comment:
`Supplement diagnostics data with response times for the top 10 resources of each cluster. These may take a while to complete`